pyproject.toml

```diff
@@ -10,15 +10,22 @@ readme = "README.md"
 python = "^3.10"
 couchdb = "^1.2"
 stopit = "^1.1.2"
-setuptools = "^66"
+setuptools = "^80"
```

Comment:
I am not sure why, but i am getting this warning now when pushing tokens to the DB (or any command that connects to DB):
```
% python push_tokens.py quick
.../lib/python3.10/site-packages/couchdb/__init__.py:15: UserWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html. The pkg_resources package is slated for removal as early as 2025-11-30. Refrain from using this package or pin to Setuptools<81.
  __version__ = __import__('pkg_resources').get_distribution('CouchDB').version
```
Could this be because of the version change in `setuptools`?